static/settings.ts

```diff
@@ -126,16 +132,21 @@ interface SliderSettings {
 class Slider extends BaseSetting {
     private readonly formatter: (number) => string;
     private display: JQuery;
+    private max: number;
+    private min: number;
 
     constructor(elem: JQuery, name: string, sliderSettings: SliderSettings) {
         super(elem, name);
 
         this.formatter = sliderSettings.formatter;
         this.display = sliderSettings.display;
 
+        this.max = sliderSettings.max || 100;
+        this.min = sliderSettings.min || 1;
+
         elem
-            .prop('max', sliderSettings.max || 100)
-            .prop('min', sliderSettings.min || 1)
+            .prop('max', this.max)
+            .prop('min', this.min)
```

Comment:
Initially the idea was to return min/max if `this.elem.val()` returned undefined, but that's nonsense, as that only happens if the `this.elem` is not matching any element, but then we could have not constructed this setting. Forgot to revert the changes but they don't hurt